docs/reference/connectors-kibana/jsm-action-type.md

```diff
@@ -0,0 +1,192 @@
+---
+navigation_title: "Jira Service Management"
+mapped_pages:
+  - https://www.elastic.co/guide/en/kibana/current/jsm-action-type.html
+applies_to:
+  stack: all
+  serverless:
+    observability: ga
+    security: ga
+---
+
+# Jira Service Management connector and action [jsm-action-type]
+
+An {{jsm}} connector enables you to create and close alerts in {{jsm}}. In particular, it uses the [{{jsm}} Integration Events API](https://developer.atlassian.com/cloud/jira/service-desk-ops/rest/v1/api-group-integration-events/#api-group-integration-events).
```

Comment:
The {{jsm}} attribute needs to be defined in our shared attributes file for it to properly resolve. According to [this](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc) error message, that's what breaking the docs build. I'll follow up on this.

Reply:
Added in [683727f](https://github.com/elastic/kibana/pull/237862/commits/683727fe2f5630810119038ee0791f7300b7bf9b).